azure-mgmt-containerregistry/src/main/java/com/azure/management/containerregistry/implementation/WebhooksClientImpl.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.azure.management.containerregistry.implementation;
+
+import com.azure.core.http.rest.PagedFlux;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.management.containerregistry.Registries;
+import com.azure.management.containerregistry.Webhook;
+import com.azure.management.containerregistry.models.WebhooksInner;
+import com.azure.management.resources.fluentcore.utils.PagedConverter;
+import reactor.core.publisher.Mono;
+
+/**
+ * Represents a webhook collection associated with a container registry.
+ */
+public class WebhooksClientImpl implements Registries.WebhooksClient {
+    private final ContainerRegistryManager containerRegistryManager;
+    private final RegistryImpl containerRegistry;
+
+    WebhooksClientImpl(ContainerRegistryManager containerRegistryManager, RegistryImpl containerRegistry) {
+        this.containerRegistryManager = containerRegistryManager;
+        this.containerRegistry = containerRegistry;
+    }
+
+    @Override
+    public Webhook get(final String resourceGroupName, final String registryName, final String webhookName) {
+        return this.getAsync(resourceGroupName, registryName, webhookName).block();
+    }
+
+    @Override
+    public Mono<Webhook> getAsync(final String resourceGroupName, final String registryName, final String webhookName) {
+        final WebhooksClientImpl self = this;
+        final WebhooksInner webhooksInner = this.containerRegistryManager.inner().webhooks();
```

Comment:
It looks not required to have final here. You can just use `return this.containerRegistryManager.inner().webhooks()....`.

Reply:
This is not a compile issue, so I haven't changed such minor issue.

Reply:
Meanwhile, I think both are OK. No one is better than the other.

Reply:
Yes, it's just minor one. My suggestion is to keep consistent coding style as other methods. You don't have to make changes here.

Reply:
Oh, yes, it is better for same style. Maybe will change it in refactor